### PR TITLE
Add summernote-classes

### PR DESCRIPTION
**Pull request by DennisSuitters**
Adds editing functionality to toggle classes and show which element is selected in the editor.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,24 +72,8 @@ This curated list is on very early stage. So, let's make it together!
...
- - [summernote-image-attributes](https://github.com/DiemenDesign/summernote-image-attributes)
-   - This is a forked and extended version of "summernote-image-title", for those that want a little more control over editable attributes for Images.
- - [summernote-image-captionit](https://github.com/DiemenDesign/summernote-image-captionit)
-   - Allows adding figure and figcaption to images, with removal when selected twice.
- - [summernote-image-shapes](https://github.com/DiemenDesign/summernote-image-shapes)
-   - Adds option with dropdown to the Image Popover to add or remove Bootstrap Image Classes for styling the look and behaviour of images.
- - [summernote-image-title](https://github.com/asiffermann/summernote-image-title)
-   - Adds a button to the image popover to edit title and alt attributes.
- - [summernote-list-styles](https://github.com/tylerecouture/summernote-list-styles)
-   - Adds a toolbar dropdown menu to change the number/bullet style of HTML lists.
```
Comment (lqez) [inline README.md:84]: It looks like you deleted other repositories, not yours. Was it intended or not?

Comment (DennisSuitters) [reply README.md:84]: No that wasn't intended. Only adding the classes link was intended.

Comment (tlartaud2) [reply README.md:84]: @DiemenDesign So, won't you revert that? :)

Comment (DennisSuitters) [reply README.md:84]: I got busy and forgot about it.

**Pull request merged**